LSP.sublime-settings

```diff
@@ -43,6 +43,14 @@
 
   // --- Diagnostics --------------------------------------------------------------------
 
+  // How to display diagnostics.
+  // Allowed options:
+  // - "highlight" - Show all diagnostic regions using highlights. Also see `diagnostics_highlight_style`.
+  // - "annotation" - Show diagnostics as annotations anchored at the right side of the corresponding line.
+  //                  When using this value it's recommended to not use the "annotation" value for the `show_code_actions`
+  //                  option as it's not possible to enforce which one is shown first then.
+  "show_diagnostics": ["highlight"],
```

Comment:
Good point. I forgot to implement that, it looks like.

Reply:
Though I don't imagine someone wanting to disable highlights as then it wouldn't not be possible to tell where the issue is on the line.

Or maybe it should be possible to set `"show_diagnostics": []` but setting `annotation` should still force highlights to show...

Reply:
I think it should be a boolean setting `"show_diagnostics_annotations": false`.

I'd imagine to use an array for the value is confusing to users, because it's not obvious from the description that you can have both at the same time (`["highlight", "annotation"]`). Also I see no real point having the "highlight" option here as well, because we already have the `"diagnostics_highlight_style": ""` option for to disable the underlines in the view. Now this setting would introduce the same again under a different setting name, and it's unclear what should happen if the settings use conflicting values.

Reply:
I guess I wanted to do a little "reset" and create some more consistency in the settings.

Your `show_diagnostics_annotations` sounds OK but we don't have `show_diagnostics_highlights` so it would be a little inconsistent.

But I'll go with your suggestion because I don't mind it and I don't want this PR to take extra year to finish :)